services/userService.js

```diff
@@ -0,0 +1,26 @@
+const bcrypt = require("bcrypt");
+
+const userDao = require("../models/userDao");
+
+const signUp = async (email, password, name, phoneNumber) => {
+  const saltRounds = 12;
+
+  const makeHash = async (password, saltRounds) => {
+    return await bcrypt.hash(password, saltRounds);
+  };
+
+  const hashedPassword = await makeHash(password, saltRounds);
```

Comment:
```suggestion
  const hashedPassword = await bcrypt.hash(password, saltRounds);
```
makeHash 함수가 지금과 같이 복잡하거나 코드가 길어지는 것이 아닌, 실행문이 단순 한줄로만 작성되어 있는 간단한 코드라면, 거기다가 다른 곳에서 재활용을 하지 않는 다면(현재 함수 내부 scope에 선언했기 때문에 다른곳에서 사용이 불가능한 부분도 있습니다.) 굳이 함수화 할 필요가 없습니다. 

따라서 위와 같이 수정되면 어떨까요?